repl/repl.go

```diff
@@ -34,7 +36,7 @@ type Querier interface {
 	Query(ctx context.Context, deps dependencies.Interface, compiler flux.Compiler) (flux.ResultIterator, error)
 }
 
-func New(q Querier) *REPL {
+func New(ctx context.Context, deps dependencies.Interface, q Querier) *REPL {
 	return &REPL{
 		scope:   values.NewScope(),
 		querier: q,
```

Comment:
should you return 
``&REPL { 
scope: values.NewScope(), 
querier: q, 
ctx: ctx,
deps: deps
}`` here?